src/main/java/com/google/sps/servlets/schedule/auth/AuthorizationServlet.java

```diff
@@ -0,0 +1,151 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.File;  // Import the File class
+import java.io.FileNotFoundException;  // Import this class to handle errors
+import java.util.Scanner; // Import the Scanner class to read text files
```

Comment:
I believe since you are not utilizing Scanner to get your information from the file, you could probably remove it.